SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -171,34 +190,18 @@ public SingularityRequestParent bounce(@ApiParam("The request ID to bounce") @Pa
       checkBadRequest(currentActiveSlaveCount >= requiredSlaveCount, "Not enough active slaves to successfully complete a bounce of request %s (minimum required: %s, current: %s). Consider deploying, or changing the slave placement strategy instead.", requestId, requiredSlaveCount, currentActiveSlaveCount);
     }
 
-    final Optional<Boolean> skipHealthchecks = bounceRequest.isPresent() ? bounceRequest.get().getSkipHealthchecks() : Optional.<Boolean> absent();
-
-    Optional<String> message = Optional.absent();
-    Optional<String> actionId = Optional.absent();
-
-    if (bounceRequest.isPresent()) {
-      actionId = bounceRequest.get().getActionId();
-      message = bounceRequest.get().getMessage();
-
-      if (bounceRequest.get().getDurationMillis().isPresent() && !actionId.isPresent()) {
```

Comment:
Good point, maybe just setup the config to validate that this must be non-zero, positive, then it's not an issue. I think you can do that with annotations. Someone could set it really high.